### ISSUE TITLE
tf.keras.utils.get_file extraction of zip not working when fname to a full path is used on Windows

### ISSUE THREAD

**Issue by rudraxx** [tensorflow/tensorflow#37269]
I am following the tensorflow image classification tutorial: https://www.tensorflow.org/tutorials/images/classification

I am unable to unzip the downloaded folder when I use a full path to a location as opposed to using just a fname as shown in the example on Windows. Have not tried on Linux, so don't know if it exists on that platform.

This is my code:
``` 
download_data_loc = os.path.join(os.path.abspath(os.path.pardir),'data\cats_and_dogs.zip')

path_to_zip = tf.keras.utils.get_file(download_data_loc,origin=_URL,extract=True, archive_format='zip')

```
I see the cats_and_dogs.zip folder in following location: C:\Users\abhat\research\Tensorflow_2.0_tutorials\data\cats_and_dogs.zip

But there isn't an unzipped folder in this location.

However, if I use,
``` 
path_to_zip = tf.keras.utils.get_file('cats_and_dogs.zip', origin=_URL, extract=True)

```
I see the zipped folder as well as the unzipped version in the right location
C:\Users\abhat\.keras\datasets\cats_and_dogs.zip

**System Info:**  
Windows 10 Pro, 64 bit OS.

**Tensorflow version:**
v2.1.0-rc2-17-ge5bf8de410 2.1.0

**Comment by ravikyram** [comment]
@rudraxx 

Can you please check unzipped folder of `cats_and_dogs.zip` will be saving in `.keras/datasets` folder. Thanks!

**Comment by rudraxx** [comment]
@ravikyram : Haven't found a great solution as yet.

Here is what I have as of now: 
1) Specify the log dir using os library,  which will give windows style path. For eg:
`logdir = os.path.join(model_folder_name, "tensorboard", "my_logs")
`
(This results in a logdir like:  ..\cnn_dog_cat_classifier\tensorboard\my_logs)

2) Launch TensorBoard from command window, NOT for the Jupyter notebook. Specify the path as a UNIX style path:
` >> tensorboard --logdir=./cnn_dog_cat_classifier/tensorboard --port=6006
`

With this, I can see the multiple runs, but for some reason I can see only the validation loss. The training loss isn't getting logged.

Please let me know if you see the same behavior.

Thanks

**Comment by rudraxx** [comment]
@vguerrero87:  Haven't found a great solution as yet.

Here is what I have as of now: 
1) Specify the log dir using os library,  which will give windows style path. For eg:
`logdir = os.path.join(model_folder_name, "tensorboard", "my_logs")
`
(This results in a logdir like:  ..\cnn_dog_cat_classifier\tensorboard\my_logs)

2) Launch TensorBoard from command window, NOT for the Jupyter notebook. Specify the path as a UNIX style path:
` >> tensorboard --logdir=./cnn_dog_cat_classifier/tensorboard --port=6006
`

With this, I can see the multiple runs, but for some reason I can see only the validation loss. The training loss isn't getting logged.

Please let me know if you see the same behavior.

Thanks

**Comment by poorghorban** [comment]
path_to_zip = tf.keras.utils.get_file('cats_and_dogs.zip', origin=_URL, extract=True)
replace :
path_to_zip = tf.keras.utils.get_file('cats_and_dogs', origin=_URL, extract=True)

**Comment by saikumarchalla** [comment]
@rudraxx  closing this issue since it was the intended behavior. Please feel free to reopen the issue if necessary.Thanks!

**Comment by Bga1972** [comment]
I am following the tensorflow text classification and I need to load a zip archive in Jupyter Notes in Windows. When I use the tf.keras.utils.get_file,, I am unable to unzip the downloaded folder, it is url as "http://...../file.zip" . 
Can you help me??
Thanks

**Comment by tamersalama** [comment]
setting the `fname=None` created the correct file with the following arguments

`data_dir = tf.keras.utils.get_file(fname=None, origin=f'file://absolute/path/to/file.zip', extract=True)`
